index.js

```diff
@@ -143,3 +150,16 @@ MultiStream.obj = streams => (
 )
 
 module.exports = MultiStream
+
+// Normalize stream destroy w/ callback.
+function destroy (stream, err, cb) {
+  if (!stream.destroy || stream.destroyed) {
+    cb(err)
+  } else {
+    const callback = once(er => cb(er || err))
```

Comment:
this wont trip up with the boolean argument from socket.on('close') ? https://nodejs.org/dist/latest-v14.x/docs/api/all.html#net_event_close_1

Reply:
it will